spec/rubocop/cop/lint/suppressed_exception_spec.rb

```diff
@@ -25,6 +25,29 @@
         end
       RUBY
     end
+
+    context 'with comment in `def`' do
+      it 'registers an offense for empty rescue' do
+        expect_offense(<<~RUBY)
+          def foo
+            do_something
+          rescue
+          ^^^^^^ Do not suppress exceptions.
+            # do nothing
+          end
+        RUBY
+      end
+
+      it 'does not register an offense for empty rescue' do
+        expect_offense(<<~RUBY)
+          def foo
+            do_something
+          rescue
+          ^^^^^^ Do not suppress exceptions.
+          end
+        RUBY
+      end
```

Comment:
In this case, you don't have a comment, so this should be inside a different context with the label:
`without comment in def` instead of with

My mistake when doing the suggestion 🙏🏻 sorry

Reply:
Ah, I was wrong 💦 Thank you for your review!